src/community/ogcapi/ogcapi-core/src/main/java/org/geoserver/ogcapi/APIConformance.java

```diff
@@ -0,0 +1,113 @@
+package org.geoserver.ogcapi;
+
+import java.util.Objects;
+
+/**
+ * Service capability or feature, identified by conformance class.
+ *
+ * OGCAPI Web Services are defined with core functionality, strictly extended with additional, optional, functionality
+ * identified by "conformance class".
+ *
+ * OGC Open Web Services can be extended using application profiles with additional, optional, functionality.
+ */
+public class APIConformance {
+
+    /**
+     * Standards approval status.
+     */
+    public enum Status {
+        APPROVED, DRAFT, INFORMAL
```

Comment:
The three statuses I know of are Approved, Draft and Proposal. However there is a lot of action happening outside of OGC and we are used to add our own vendor extensions too
With basically no-one in our community having time to also participate in working groups I would probably address the representation of the status as two enums:

Stability: finalized, draft (whether the conformance is yet subject to change)
Source: OGC, Vendor (whether it's an official standard or coming from outside)

That said... I believe in terms of defaults and user concerns, only the first one is relevant... can one rely on the behavior of a given conformance class long term, or not?  That basically reduces what we need to communicate to a boolean. The source can be an interesting informative notion, but that's it.

Reply:
I also wish to have an "ogc only" mode if *strict* is in play? I guess given the building block approach it should not technically matter but ...